common/audio/audio_formats.cpp

```diff
@@ -1,6 +1,7 @@
 #include "audio_formats.h"
 #include "common/util/BinaryWriter.h"
 #include "third-party/fmt/core.h"
+#include <common/log/log.h>
```

Comment:
I think we should use `"common/log/log.h"` instead of the angle brackets in all these files. The angle brackets are usually used for system headers that need to search on some weird out-of-project path.